src/test/java/com/example/postgresdemo/controller/QuestionControllerTest.java

```diff
@@ -0,0 +1,192 @@
+package com.example.postgresdemo.controller;
+
+import com.example.postgresdemo.model.Question;
+import com.example.postgresdemo.repository.QuestionRepository;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.web.context.WebApplicationContext;
+
+
+import java.nio.CharBuffer;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.jsonPath;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+
+@AutoConfigureMockMvc
+public class QuestionControllerTest {
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private QuestionController questionController;
```

Comment:
I think `QuestionController ` and `WebApplicationContext ` fields can be removed as they aren't used in any test